content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -203,23 +209,24 @@ this can be changed using the namespace selector located in the navigation menu.
 
 Dashboard shows most Kubernetes object kinds and groups them in a few menu categories.
 
-#### Admin Overview
-For cluster and namespace administrators, Dashboard lists Nodes, Namespaces and Persistent Volumes and has detail views for them.
+#### Admin overview
+
+For cluster and namespace administrators, Dashboard lists Nodes, Namespaces and PersistentVolumes and has detail views for them.
```

Comment:
I think "and has detail views for each of them" may sound better here. This is just a small nit though. Feel free to disregard.